build/packer/ansible/roles/docker-setup/templates/pmm2.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=PMM2 Docker container
```

Comment:
```suggestion
Description=PMM Docker container
```

Reply:
We're using following:
Description=pmm-server